### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by NathanBnm**
There were a few strings that were missing in translation files. Added a comment to in order to help translators.

### CODE REVIEW

**Comment by NathanBnm** [comment]
I don't know what's happening, not only the `TRANSLATORS` labelled comments are injected into the translation files. @ryonakano could you help?

File: po/meson.build [hunk 1]
```diff
@@ -2,7 +2,7 @@ i18n.gettext(meson.project_name(),
     args: [
         '--directory=' + meson.source_root(),
         '--from-code=UTF-8',
-        '-cTRANSRATORS'
+        '-cTRANSLATORS'
```
Comment (ryonakano) [inline po/meson.build:5]:
```suggestion
        '-cTRANSLATORS:'
```

This should resolve the issue :)

**Comment by NathanBnm** [comment]
@ryonakano unfortunately it's still not working :confused:

File: src/Widgets/TimerTimeWidget.vala [hunk 2]
```diff
@@ -99,6 +99,7 @@ namespace Hourglass.Widgets {
             chooser_grid.attach (new Spacer.w_hexpand (), 0, 0, 1, 1);
             chooser_grid.attach (hour_chooser, 1, 0, 1, 1);
 
+            // TRANSLATORS: Time separator, e.g. 00:00:00
```
Comment (ryonakano) [inline src/Widgets/TimerTimeWidget.vala:102]:
```suggestion
            //TRANSLATORS: Time separator, e.g. 00:00:00
```

Does this work?

Comment (NathanBnm) [reply src/Widgets/TimerTimeWidget.vala:102]: @ryonakano unfortunately not, every comment is still extracted. Don't know why :confused:

Comment (ryonakano) [reply src/Widgets/TimerTimeWidget.vala:102]: Hmm, strange, this removes all unwanted comments on my side :thinking:

Comment (NathanBnm) [reply src/Widgets/TimerTimeWidget.vala]: @ryonakano fixed now. I had to remove the colon you made me add in the previous suggestion AND to apply this suggestion too.

I also reconfigured the build before trying again, maybe there was a sort of cache issue.

**Pull request merged**